Css/style.css

```diff
@@ -0,0 +1,84 @@
+*{
+    padding: 0;
+    margin: 0;
+    outline: none;
+}
```

Comment:
**General comment :** 

It will be better to separate sections at `CSS` & `HTML` by writing a descriptive comment for each part or section.